src/lib/execPromise.ts

```diff
@@ -37,33 +41,40 @@ export type ExecPromiseArg = {
   printOutput?: boolean | undefined;
 };
 
-export const execPromise = ({
+export const execPromise = async ({
   command,
   args,
   flags,
   options,
   spinnerMessage,
   timeout,
-  printOutput = false,
+  printOutput: printOutputArg = false,
 }: ExecPromiseArg): Promise<string> => {
+  const printOutput = isInteractive ? printOutputArg : false;
   const flagArgs = flags === undefined ? [] : flagsToArgs(flags);
   const allArgs = [...(args ?? []), ...flagArgs];
   const fullCommand = [command, ...allArgs].join(" ");
 
-  const commandToDisplay = IS_DEVELOPMENT
-    ? fullCommand
-    : fullCommand.replace(
-        /([\S\s]*--sk ).*([\S\s]*)/,
-        "$1<SECRET_KEY_IS_HIDDEN>$2"
-      );
+  const getCommandFailedMessage = (code: number | null = null) => {
+    return `Command: ${color.yellow(
+      IS_DEVELOPMENT
```

Comment:
How `IS_DEVELOPMENT` is set?

Reply:
if you use ./bin/dev then it is development. Otherwise it's not
It's just so while developing you can see the actual args and in prod secret keys are not leaked
I think now that we don't use aqua cli this part can even be removed
I will do it